specs/airr-schema.yaml

```diff
@@ -1847,19 +1847,38 @@ Rearrangement:
                 subset: data (processed sequence)
                 name: V(D)J germline reference database
 
+# 1-to-n relationship between a CloneSet and the Repertoires it is derived from
+# Question: Do we want to include a direct link to `Subject` in this object?
+CloneSet:
+    discriminator: AIRR
+    type: object
+    properties:
+        clone_set_id:
+            type: string
+            description: Identifier for the clone set object.
+        repertoire_list: #did I do this correctly?
+            type: array
+            description: List of repertoires that were processed to produce clone set
+            items:
+                - $ref: '#/Repertoire'
+        data_processing_id:
+            type: string
+            description: Indentifier for the data processing object
+            $ref: '#/DataProcessing'
+
+
 # A unique clone object that has been constructed from a single clone processing.
-# The information provided here is common among or aggregated across all repertoires
-# in the clone processing.
 Clone:
     discriminator: AIRR
     type: object
     properties:
         clone_id:
             type: string
             description: Identifier for the clone.
-        data_processing_id:
+        clone_set_id:
```

Comment:
Is there any benefit to having this relationship referenced the other direction (directly as opposed to through `repertoire_list`). That is, in addition to (or instead of) having a clone_set_id in a Clone, we would have a list of clone_id in a CloneSet.

Or will a CloneSet always be composed of sets of Repertoires and so `repertoire_list` is enough to describe the relationship between CloneSet and Clone?

Reply:
The way I have it set up, you need the `clone_set_id` in `Clone` in order to be able to get back to the `DataProcessing`, but I think that you're right, `CloneSet` should probably have a list of `clone_id`s in it.